modules/puppetca/puppetca_api.rb

```diff
@@ -2,14 +2,18 @@
 
 module Proxy::PuppetCa
   class Api < ::Sinatra::Base
+    extend Proxy::PuppetCa::DependencyInjection
+    inject_attr :cert_manager, :cert_manager
+    inject_attr :autosigner, :autosigner
```

Comment:
How about just `autosign`?

Reply:
I like to refer to objects with nouns, makes it better to read imho ^^

Reply:
Ok, works for me.